DESCRIPTION

```diff
@@ -42,7 +42,8 @@ Suggests:
     gt,
     dplyr,
     knitr,
-    roxyglobals
+    roxyglobals,
+    rstan
```

Comment:
I ideally we want to move away from needing to use `rstan` and instead depend on `posteriors` for anything we can't get from `brms`